chore: document Android haptic feedback (#556)
keymanapp/help.keyman.com: developer/engine/android/15.0/KMManager/getHapticFeedback.md

---
title: KMManager.getHapticFeedback()
---

## Summary

> The **getHapticFeedback()** method returns whether the keyboard vibrates as the user types.

Reviewer: ```suggestion
The **getHapticFeedback()** method returns whether the device vibrates as the user types.
```

## Syntax
```java
KMManager.getHapticFeedback()
```

### Returns

> Returns `true` if the keyboard vibrates as the user types, `false` otherwise.

Reviewer: ```suggestion
Returns `true` if the device vibrates as the user types, `false` otherwise.
```

## Description

> Use this method to check if the keyboard provides haptic feedback. Default is `false`.

Reviewer: ```suggestion
Use this method to check if the device is currently configured to provide haptic feedback. Default is `false`.
```

## History

> Added syntax in Keyman Engine for Android 15.0.

Reviewer: ```suggestion
Keyman Engine for Android 15.0: New function.
```
